packages/base-zone/src/index.js

```diff
@@ -6,4 +6,3 @@ export * from './exports.js';
 // Utilities for creating zones.
 export * from './make-once.js';
 export * from './keys.js';
-export * from './is-passable.js';
```

Comment:
Can you leave this line intact?  If you really want to remove it, please mark a conventional commit that does only that as a breaking change.

Reply:
Done.